packages/firestore/src/local/indexeddb_remote_document_cache.ts

```diff
@@ -206,46 +218,72 @@ export class IndexedDbRemoteDocumentCache implements RemoteDocumentCache {
   ): PersistencePromise<DocumentSizeEntries> {
     let results = nullableMaybeDocumentMap();
     let sizeMap = new SortedMap<DocumentKey, number>(DocumentKey.comparator);
```

Comment:
Sorry, now that getEntries() is no longer a wrapper around getSizedEntries(), can we drop sizeMap and have this be new SortedMap<DocumentKey, DocumentSizeEntry|null> ?

Reply:
First, I don't feel strongly about this. The reason I set it up that way is so that `getEntries` in `RemoteDocumentChangeBuffer` can return the `MaybeDocumentMap` directly. If this were a sorted map of `DocumentSizeEntry`s, then `getEntries` would have to create a new `MaybeDocumentMap` and fill it with just `MaybeDocument`s.

It's probably not a big deal, so if you think code clarity is more important here, I'll do the change.

Reply:
Oh, sorry! I was confused.  I thought it was just so the old getEntries() implementation could return the MaybeDocumentMap directly.  But I see now that RemoteDocumentChangeBuffer.getEntries() ends up calling getSizedEntries() and using the sizes and also passing the MaybeDocumentMap straight through.  So it needs both, and the way it's structured right now makes sense.

So nevermind. Please keep it the way it is.